<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>amedicament</class>
 <widget class="QMainWindow" name="amedicament">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des médicaments</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_rechercher">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Chercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Déconnexion</string>
    </property>
   </widget>
   <widget class="QTableView" name="tablemedicament_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>891</width>
      <height>641</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tablemedicament">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomP_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;text-decoration:none;</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>250</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233,248,255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">;</string>
     </property>
     <property name="text">
      <string>Référence:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Nom du produit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>nombre de médicament:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ref">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nb">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);text-decoration:none;</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercherP">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nomP">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_tri">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Trier Liste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));
background-color: rgb(234, 234, 234);</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_imprimer">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Imprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supp">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_commander">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Commander </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Chercher médicaments en saisissant le nom du produit recherché:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 123, 180);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(234, 234, 234);
color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0.512712 rgba(70, 156, 208, 255));</string>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <zorder>label_10</zorder>
   <zorder>pushButton_rechercher</zorder>
   <zorder>pushButton</zorder>
   <zorder>tablemedicament_2</zorder>
   <zorder>tablemedicament</zorder>
   <zorder>lineEdit_nomP_3</zorder>
   <zorder>frame</zorder>
   <zorder>pushButton_tri</zorder>
   <zorder>pushButton_ok</zorder>
   <zorder>pushButton_imprimer</zorder>
   <zorder>pushButton_supp</zorder>
   <zorder>pushButton_commander</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_modifier</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
